Islandora-CLAW/islandora
# method check for remote videos

# What does this Pull Request do?

Currently if we add a remote video media type Islandora whitescreens when it tries to use the non-existent referencedEntities() method.  This small change just checks for the method's existence before attempting to run.

# What's new?
One new check before execution.

# How should this be tested?
Add a remote video as a media. (You'll have to add mediaOf and mimeType as existing fields to the mediaType).

Try to display that media and you'll see a whitescreen.
Apply this pull, and you should see your embedded video.


# Interested parties
@Islandora-CLAW/committers
